src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -416,30 +416,31 @@ void AtlasEngine::_waitUntilCanRender() noexcept
 void AtlasEngine::_present()
 {
     // Present1() dislikes being called with an empty dirty rect.
-    if (!_p.dirtyRectInPx)

```

Comment:
So, this is the LOAD BEARING change in this PR, the rest is incidental?

Reply:
Yeah, sort of. Due to the type change a couple more changes are necessary unfortunately.